app/controllers/finance/ecf/payment_breakdowns_controller.rb

```diff
@@ -11,14 +11,64 @@ def show
           contract: @ecf_lead_provider.call_off_contract,
           event_type: :started,
         )
-        @cutoff_date = "On #{helpers.cutoff_date}"
```

Comment:
I've removed as many helpers as I reasonably can, there are probably more for another time. I don't think putting view helpers in controllers in a great idea.